src/web_video_server.cpp

```diff
@@ -117,32 +127,26 @@ WebVideoServer::WebVideoServer(rclcpp::Node::SharedPtr &nh, rclcpp::Node::Shared
   }
```

Comment:
This print was removed, but I found it really useful to check that everything is launching correctly
```suggestion
    RCLCPP_INFO(nh_->get_logger(), "Waiting For connections on %s:%d", address_.c_str(), port_);
  }
```